benefits/static/css/admin/overrides.css

```diff
@@ -0,0 +1,22 @@
+.theme-toggle {
+  display: none;
+}
+
+#content-start h1 {
+  font-size: 1rem;
+  font-weight: 700;
+  margin: 12px 0 20px 0;
+}
+
+#content-related h3,
+#changelist-filter-extra-actions h3 {
+  font-size: 0.875rem;
+}
+
+table {
+  caption-side: top;
+}
+
+#user-tools {
+  letter-spacing: 0;
+}
```

Comment:
Why have another CSS file? This file is intended to be one-off styles _only_ for Django CSS. This file is not used on `agency-base` or `error-base`. I wanted to keep these one-off overrides separate from any custom flows we write.